app/services/metrics/weekly/review_turnaround/per_user_project.rb

```diff
@@ -0,0 +1,52 @@
+module Metrics
+  module Weekly
+    module ReviewTurnaround
+      class PerUserProject < BaseService
+        def call
+          process
+        end
+
+        private
+
+        def process
+          daily_metrics.find_each do |daily_metric|
+            weekly_metric = weekly_of(daily_metric.ownable)
+            if weekly_metric.present?
+              next weekly_metric.update!(
+                value: (daily_metric.value + weekly_metric.value) / current_time.wday
+              )
+            end
+
+            create_weekly(daily_metric)
+          end
+        end
+
+        def create_weekly(metric)
+          Metric.create!(
+            value: metric.value,
+            interval: :weekly,
+            name: :review_turnaround,
+            ownable: metric.ownable,
```

Comment:
I think it would be faster to just do:

```
ownable_type: metric.ownable_type,
ownable_id: metric.ownable_id
```

That way we don't have to query the db and load the object in memory. Maybe Rails already does this and doesn't do the query, I'm not sure.